assets/js/opensuse-theme.js

```diff
@@ -133,8 +133,9 @@ function osMoreInformation(os) {
 
   moreInfoOpened = true
   //get the height of the main container
-  containerHeight = $('#opensuse-os').outerHeight()
-
+  containerHeight = $('#opensuse-os').outerHeight();
+  window.location.hash = '0';
```

Comment:
That was completely understood. My point was to make it even better by having an actual URL. But its fine, I will create a new card for that to work on it this friday ;)